<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sliceFilterPropForm</class>
 <widget class="QWidget" name="sliceFilterPropForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slice Property</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>plane</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>show plane</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>origin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="m_planeOrigin_lineEdit1"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="m_planeOrigin_lineEdit2"/>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="m_planeOrigin_lineEdit3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>normal</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="m_planeNormal_lineEdit1"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="m_planeNormal_lineEdit2"/>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="m_planeNormal_lineEdit3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="xNormal_btn">
          <property name="text">
           <string>X Normal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="yNormal_btn">
          <property name="text">
           <string>Y Normal</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="zNormal_btn">
          <property name="text">
           <string>Z Normal</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="cameraNormal_btn">
          <property name="text">
           <string>Camera Normal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="insideOut_checkBox">
     <property name="text">
      <string>Inside out</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
